.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
   check-permission:
     name: Check permission
     if: contains(github.ref, 'refs/heads/master')
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
There is no reason to change from `ubuntu-latest`. Please leave these as-is to stay current.